app/controllers/api/databases.php

```diff
@@ -97,6 +100,14 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
         throw new Exception(Exception::ATTRIBUTE_DEFAULT_UNSUPPORTED, 'Cannot set default value for array attributes');
     }
 
+    if ($type === Database::VAR_RELATIONSHIP) {
+        $options['side'] = Database::RELATION_SIDE_PARENT;
```

Comment:
```suggestion
        $options['side'] = Database::RELATION_SIDE_PARENT;
```

Isn't it better to use `DIRECTION` instead of `SIDE` ?

Reply:
I think side makes more sense for parent/child; direction could be confusing with one-way/two-way as well